catroid/src/main/java/org/catrobat/catroid/ui/dialogs/OrientationDialog.java

```diff
@@ -135,12 +136,19 @@ protected void handleOkButtonClick() {
 		dismiss();
 	}
 
-	private void loadStageProject() {
-		String templateZipFileName =
-				createLandscapeProject ? templateData.landscapeZipFileName : templateData.portraitZipFileName;
+	private void downloadTemplateFile() {
+		String zipFileUrl = baseUrlForTemplates;
+		zipFileUrl += createLandscapeProject ? templateData.getLandscape() : templateData.getPortrait();
+
+		DownloadTemplateTask downloadTemplateTask = new DownloadTemplateTask(getActivity(), templateData, zipFileUrl);
```

Comment:
There is also a getActivity()? Then why you have to put the activity as a member?